src/lib/mina.ts

```diff
@@ -217,11 +222,13 @@ function LocalBlockchain({
 
   const ledger = Ledger.create([]);
 
+  let networkState = defaultNetworkState();
+
   function addAccount(pk: PublicKey, balance: string) {
     ledger.addAccount(pk, balance);
   }
 
-  let testAccounts = [];
+  let testAccounts: any[] = [];
```

Comment:
Is there a way to type this? Not sure what the accounts data looks like but it would be a nice touch! Could we define a type in TS and just cast it over?